nodejs/user-interface/modals/__tests__/new-task.test.js

```diff
@@ -123,5 +152,5 @@ test('Returns blocks for the new task modal if a prefill is provided', () => {
     ],
     type: 'modal',
   };
-  expect(newTask(taskTitle)).toBe(JSON.stringify(expected));
+  expect(newTask(taskTitle, testUserID)).toBe(JSON.stringify(expected));
```

Comment:
The tests needed to be updated since we'd changed the behaviour. We should always add new tests when we introduce new behavior.